integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/transactions/CrowdNodeWithdrawalReceivedTx.kt

```diff
@@ -46,6 +52,11 @@ class CrowdNodeWithdrawalReceivedTx(
         return false
     }
 
+    fun and(txFilter: TransactionFilter): CrowdNodeWithdrawalReceivedTx {
+        joinedFilters.add(txFilter)
+        return this
+    }
```

Comment:
If this proves useful, we might want to have a base class that supports combining filters for all descendants.